web/src/client/users.js

```diff
@@ -122,6 +124,24 @@ export default class UsersClient {
     const result = await proxy.SetRootSSHKey(key);
     return result === 0;
   }
+
+  /**
+   * Register a callback to run when properties in the Users object change
+   *
+   * @param {function} handler - callback function
+   */
+  onUsersChange(handler) {
+    return this.onObjectChanged(USERS_PATH, changes => {
+      if (changes.RootPasswordSet) {
```

Comment:
I wonder if we can get more than one change simultaneously (for instance, using the CLI).

Reply:
is that really possible?, wouldn't be two different events? I mean we could have both for example with different clients using the CLI.. but would not expect to be a problem...

Reply:
`changes` is plural, so I expect you can have more than one change. Let's say you have a "PropertiesChanged" with many properties... I am not saying today, with our current UI, it is possible. But programmatically, I would expect it is possible.

Reply:
Yep, not with the current UI and CLI implementation but it could happen.